lib/api/wpc_proto.h

```diff
@@ -157,9 +157,11 @@ app_proto_res_e WPC_Proto_register_for_event_status(onEventStatus_cb_f onEventSt
 /**
  * \brief        Get current event Status. It is mainly required at boot time of gateway
  *               to publish Gateway status (and no event status are generated yet)
- * \param[in]    online
+ * \param[in]    gw_online
  *               If true, get the "ONLINE" current status. If false, get the "OFFLINE" status
  *               to be published as last will topic
+ * \param[in]    sink_online
+ *               If true, one sink config is attached, if false, none.
```

Comment:
Maybe you could add a note that false is in case of communication issue with sink and to notify the situation on broker.
But it is not a normal case